Formula/rustup.rb

```diff
@@ -0,0 +1,40 @@
+class Rustup < Formula
+  desc "The Rust toolchain installer"
+  homepage "https://github.com/rust-lang-nursery/rustup.rs"
+
+  url "https://github.com/rust-lang-nursery/rustup.rs/archive/1.3.0.tar.gz"
+  sha256 "c0ca06b70104fed8f1de5a6f5ecfd8478e8bc03f15add8d7896b86b3b15e81e3"
+
+  depends_on "rust" => :build
+
+  def install
+    cargo_home = buildpath/"cargo_home"
+    cargo_home.mkpath
+    ENV["CARGO_HOME"] = cargo_home
+
+    system "cargo", "build", "--release", "--verbose"
+
+    bin.install buildpath/"target/release/rustup-init"
+  end
+
+  def caveats
+    <<-EOS.undent
+      rustup allows you to customize the location of your rust and cargo
+      installations by setting the CARGO_HOME and RUSTUP_HOME environment
+      variables.
+
+      CARGO_HOME defaults to ~/.cargo/
+      RUSTUP_HOME defaults to ~/.multirust/
+
+      To get started, run:
+
+        rustup-init
+
+    EOS
+  end
+
+  test do
+    assert_equal "rustup-init 1.3.0 ( )",
```

Comment:
Embed `#{version}` so this doesn't need updated for every release. Alternatively, use `assert_match` and don't match on the version at all (and remove the `.strip`)